automatic/keepass.portable/keepass.portable.nuspec

```diff
@@ -54,6 +54,18 @@ KeePass currently is available in [two different editions](http://keepass.info/c
 KeePass 1.x may be obtained via chocolatey package [keepass-classic](https://chocolatey.org/packages/keepass-classic).
 KeePass 2.x may be obtained via this package.
 
+### Package Parameters
+
+The following package parameter can be set:
+
+* `/target:` specify a custom extract location as a full path - defaults to choco folder.
+
+To specify a custom location for the portable files use:
+
+```PowerShell
+choco install keepass.portable --params="'/target:C:\Users\username\OneDrive\KeePass'"
+```
+
```

Comment:
No need to update the .nuspec as it will take it's description from the README.md file.